views/homework/evaluation.hbs

```diff
@@ -78,13 +71,7 @@
                         <p>{{$t "homework.text.noEvaluationTextAvailable" }}</p>
                     {{/if}}
                 </div>
-                {{#ifneq 0 (arrayLength this.submission.gradeFileIds)}}
-                    {{#embed "homework/components/submission_files"
-                        fileIds=../submission.gradeFileIds
-                        grading=true
-                        deletionAllowed=false
-                        submission=../submission}}{{/embed}}
-                {{/ifneq}}
+                {{#embed "lib/components/files-storage/files" this.submission.gradeFiles}}{{/embed}}
```

Comment:
Das müsste weiter oben in das `{{#if this.submission.submitted}}` bzw. `{{#if @root.isTeacher}}` oder? Wird aktuell auch dem Schüler angezeigt,